f64/at_least.go

```diff
@@ -1,10 +1,10 @@
-package float64validator
+package f64
```

Comment:
I know we discussed this yesterday, but I'm wondering whether we do want the "validator" text in the package name. Reason being, when we're adding `Validators` to `Attributes` for instance, with the original naming, we're going to have something like:
```go
Validators: []tfsdk.AttributeValidator{
	int64Validator.AtLeast(1),
```

Whereas, the new package naming will give us:
```go
Validators: []tfsdk.AttributeValidator{
	i64.AtLeast(1),
```

This is certainly more succinct, but are we losing useful information about what `i64` is?

Reply:
Given their signature, it seems clear to me at least that those are validators.

And for the type info, I admit I didn't invent this naming: I first [saw it in Rust](https://doc.rust-lang.org/rust-by-example/primitives.html#scalar-types).

Reply:
I even tried to use `int64`, `float64` and `string` in the name btw, but `predeclared` linter started complaining about that identifier being used.

Reply:
Fair point. Ok, I'm happy to go with this if there's consensus.

Reply:
I spoke with @bflad and he has a very compelling case to avoid naming the packages by type: name clashing when importing.

If/when we create a new package for `AttributePlanModifier`s etc, as we will likely be having the same "by type" separation, we could end up with 2 packages, under 2 different repo, with the same name. Forcing imports to use _aliasing_. That is indeed annoying.

So I'm going to give up, and just move `diag` under `helpers/validatordiag`, as more `helpers` are incoming.